server/pbench/bin/pbench-unpack-tarballs

```diff
@@ -159,12 +159,11 @@ while read size result ;do
         popd > /dev/null 2>&1
 
 	basedir=$(dirname $link)
-        prefixfile=$basedir/prefix.$resultname
+        # pbench-dispatch has already moved it to the .prefix subdir
+        prefixfile=$basedir/./prefix/prefix.$resultname
```

Comment:
There is an unnecessary backslash in the path; which makes it a wrong path. It should be prefixfile=$basedir/.prefix/prefix.$resultname

Reply:
RIght - thanks!